packages/round-manager/src/features/round/ViewRoundPage.tsx

```diff
@@ -65,6 +65,11 @@ export default function ViewRoundPage() {
       setRoundExists(!!round);
 
       if (round) {
+        /* During development, give frontend access to all rounds */
+        if (process.env.REACT_APP_IGNORE_FRONTEND_CHECKS) {
```

Comment:
Should we note this new env var and use it elsewhere,too?

Reply:
Noted in .env.example. Use elsewhere would be great but oos for this pr.